metastats-domain/src/main/groovy/life/qbic/metastats/exceptions/InvalidProjectStructureException.groovy

```diff
@@ -0,0 +1,26 @@
+package life.qbic.metastats.exceptions
+
+/**
+ * Invalid project structures thrown an InvalidProjectStructureException
+ *
+ * If an imported project does not follow the data model as described in doc/Openbis_Projectstructure.md this error needs to be thrown
+ *
+ * @since: 1.0
+ * @author: Jennifer Bödker
+ *
+ */
+class InvalidProjectStructureException extends IllegalArgumentException {
+
+    /**
+     * Creates an IllegalFileType exception with an error message
+     * @param error Content of error message
+     */
+    InvalidProjectStructureException(String error){
+        super(error)
+    }
+
+    @Override
+    String getMessage() {
+        return super.getMessage()
+    }
```

Comment:
Since you do not change the behaviour of the superclass method I would remove it here.